app/views/service-start/index.njk

```diff
@@ -54,7 +54,7 @@
       text: "Start now"
     }) }}
 
-    <p>You can also <a id='downloadAllLink' href="/">download all scheme payment data (.CSV, 345KB)</a> for the:</p>
+    <p>You can also <a id='downloadAllLink' href="/downloadall">download all scheme payment data (.CSV, 345KB)</a> for the:</p>
```

Comment:
Can you please confirm if that file size in the text is correct? I know there were a couple of mentions of the size being different if I am not mistaken.

Reply:
The file is not changed yet, this comes from the markup. Sam is preparing the actual file , once he provides the size details , we should update it here. But that will be later.